src/app/pagamentoplanos/component/pagamentoplanos.component.ts

```diff
@@ -7,9 +9,119 @@ import { Component, OnInit } from '@angular/core';
 })
 export class PagamentoplanosComponent implements OnInit {
 
-  constructor() { }
+  constructor(
+    private scriptService: ExternalService,
+    private pagamentoSerice: PagamentoplanosService
+  ) { }
 
   ngOnInit(): void {
+    this.scriptService.load('MercadoPago').then(data => {
+  }).catch(error => console.log(error));
+    //@ts-ignore
+    const mp = new MercadoPago('YOUR_PUBLIC_KEY');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PadawansTeam/artmais-frontend/pullRequest?prid=8398929)
```suggestion
    const mp = new MercadoPago("YOUR_PUBLIC_KEY");
```